save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProject.kt

```diff
@@ -22,14 +22,13 @@ import java.io.File
 
 /**
  * A Spring controller for git project downloading
+ *
+ * @property configProperties config properties
  */
 @RestController
-class DownloadProject {
+class DownloadProject(val configProperties: ConfigProperties) {
```

Comment:
Should it be `@autowired`?

Reply:
No, it shouldn't. We can omit the `@autowired`